### ISSUE TITLE
Problem accessing Occurrences on map

### ISSUE THREAD

**Issue by skybristol** [iobis/obis-issues#10]
I'm getting a problem for any taxon going to the Occurrences map. I've tried with Chrome and Firefox on Mac. The map zooms to the Mediterranean and never shows me any points. When I zoom out a couple of levels, the map goes back to the "Distribution" map.

**Comment by skybristol** [comment]
I understand that this is not a bog but a feature of how the system is attempting to deal with point data. The point data will only show up at a certain zoom level, and it will go back to showing the data density chloropleth grids once zoomed out. This is still a confusing situation on the actual map interface and some other method for dealing with the point data should be explored.

**Comment by pieterprovoost** [comment]
I agree this is confusing, and I'm looking into a solution for displaying large amounts of points. Perhaps raster tiles with UTFGrid.

**Comment by skybristol** [comment]
You might try the PointStacker method with Geoserver as one option. That's a clustering method vs a grid method. Both can be useful.
